### ISSUE TITLE
Accessibility: MAS13: Zoom: LHS pane contents are missing when zoomed to 200%

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#25]
#A11yMAS
#MAS13
#Wipro
#vscode-website

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Browser-Edge
Edge Version-Microsoft Edge 38.14393.461.0

**Repro Steps:**
1.Open https://code.visualstudio.com/docs in edge, zoom the page to 200%.
2.Navigate to the LHS pane below the header of the page.
3.Observe the LHS pane.[screenshot]

**Note: same issue observed for below links**
https://code.visualstudio.com/docs/introvideos/overview
https://code.visualstudio.com/docs/editor/codebasics
https://code.visualstudio.com/docs/customization/userandworkspace
https://code.visualstudio.com/docs/languages/overview
https://code.visualstudio.com/docs/runtimes/nodejs

**Actual Result:**
Observe the LHS pane of the page  ,the overview contents are missing when zoomed to 200% but clearly visible when zoomed to normal (100%).

**Resolution: 1366x768**

**Note:** Missing  overview contents are 
"Intro Videos","Setup","Editor","Customization", Languages",
"Runtimes","Extension Authoring","Extensibility Reference","Tools".[screenshot]

**Expected Result:**
The contents should not be missed when zoomed to 200%.

**User Impact:**
Users with low vision and people who need cognitive assistance will not be able identify the missing interactive elements.

**Screenshots:**
![missing lhs pane contents when zoomed to 200prcnt](https://cloud.githubusercontent.com/assets/24825441/21709372/85b32c82-d406-11e6-80e7-9525009a2f51.png)
![visible contents in lhs pane_normal view](https://cloud.githubusercontent.com/assets/24825441/21709373/871b16c0-d406-11e6-8106-ebd54eaba5d7.png)

**Comment by chrisdias** [comment]
Related to https://github.com/wiproaccess/vscode-website/issues/23

**Comment by chrisdias** [comment]
Won't fix. The LHS is available in the TOPICS dropdown when at 200%.
